### ISSUE TITLE
How should we deal with typeclasses that need particular instances of other typeclasses to be lawful?

### ISSUE THREAD

**Issue by johnynek** [non/cats#805]
See: https://github.com/non/algebra/pull/126#discussion_r45569352

`Group[Map[K, V]]` needs a vetor space style `Eq[Map[K, V]]` that considers `eqv(Some(0), None)`

**Comment by rklaehn** [comment]
Not 100% related to this question: I am not sure if you followed the discussion. But I think it would probably be best to leave instances out of cats-kernel for now to reduce the potential for disagreement. Do you agree?

**Comment by johnynek** [comment]
Yes, I just wanted to add an issue to not lose track of this design problem.

Agree we should leave instances out for now.
